README.md

```diff
@@ -101,6 +101,30 @@ Responses have JSONs identical to the original api <https://us.incognia.com>, **
 }
 ```
 
+## Exception handling
+
+Every method call can throw `IncogniaAPIError` and `IncogniaError`.
+
+`IncogniaAPIError` is thrown when the API returned an unexpected http status code. You can retrieve it by calling the `statusCode` property, along with the `payload` property, which returns the API response payload that might include additional details.
+
+`IncogniaError` represents unknown errors, like serialization/deserialization errors.
+
+```js
+const { IncogniaAPI, IncogniaAPIError } = require('@incognia/api')
+
+try {
+  const loginAssessment = await incogniaAPI.registerLoginAssessment({
+    installationId: 'installation_id',
+    accountId: 'account_id'
+  })
+} catch (error) {
+  if (error instanceof IncogniaAPIError) {
+    console.log(error.statusCode)
+    console.log(error.payload)
+  }
+}
```

Comment:
Maybe this works to catch this specific error, wdyt?

```suggestion
} catch (error if error instanceof IncogniaAPIError) {
    console.log(error.statusCode)
    console.log(error.payload)
}
```

Reply:
Cool! 
d4e46bcfbddcac62179957a8173ff46c128b8fab

Reply:
Ops, I'm gonna revert it.

> Non-standard
This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.

Reply:
:cry: